docs/reset-password.md

```diff
@@ -0,0 +1,54 @@
+# Reset TechPass password
+
+If you are a vendor and if your TechPass domain is ```techpass.gov.sg```, you may reset your TechPass password according to the [Azure Password Policy](https://learn.microsoft.com/en-us/azure/active-directory/authentication/concept-sspr-policy#administrator-password-policy-differences) or if your TechPass account gets locked.
```

Comment:
My suggestion is "If you are a vendor with login domain `techpass.gov.sg`".

Reply:
@JLeeChan , Can it be "If you are a vendor or contractor and if your TechPass logon domain is  ```techpass.gov.sg```, you may reset your TechPass password..?

Reply:
i was trying to avoid the **if .. and if ..**, i thought **if .. and ..** looks better.

Reply:
got it!

Reply:
fixed it.